models/activities/repo_activity.go

```diff
@@ -172,7 +173,7 @@ func (stats *ActivityStats) MergedPRPerc() int {
 
 // ActiveIssueCount returns total active issue count
```

Comment:
The comment needs to be changed.

Reply:
Not sure I follow. We are returning the total number of active issues (actually, before we were not).